src/hmem_ze.c

```diff
@@ -952,6 +999,35 @@ int *ze_hmem_get_dev_fds(int *nfds)
 	return dev_fds;
 }
 
+int ze_hmem_host_register(void *ptr, size_t size)
+{
+	ze_result_t ze_ret;
+
+	if (host_reg_enabled) {
+		ze_ret = ofi_zexDriverImportExternalPointer(driver, ptr, size);
+		if (ze_ret) {
+			FI_WARN(&core_prov, FI_LOG_CORE,
+				"Failed to import host memory: ptr %p size %zd",
+				ptr, size);
+		}
+	}
+	return FI_SUCCESS;
```

Comment:
Is it okay to always return success even if we fail or the feature is disabled?

Reply:
Yes, on failure or disabled it acts like no-op.